docs/how_to/use_toggles.rst

```diff
@@ -0,0 +1,65 @@
+***************************************
+How to: Use toggles
+***************************************
+
+Calling toggles
+==================
+
+Unlike the underlying waffle library which looks up waffles by name,
+toggles should be imported and used directly:
+
+.. code:: python
+
+    from whereever import FLAG_FOR_STUFF, COURSE_FLAG_FOR_STUFF
+
+    FLAG_FOR_STUFF.is_enabled()
+    COURSE_FLAG_FOR_STUFF.is_enabled('course-key-123')
+
+You can get at the state of these toggles by name via the waffle library
+and it will appear to work fine in simple testing. Don't be fooled.
```

Comment:
because toggles are built on the waffle library I think you can end-run them using base waffles by name and get the data. Maybe only bad people like me think of these things. :)

Reply:
We have linting that helps stop you from using waffle directly.

Reply:
in this case I was going from one library to another rather than directly in platform, so I'm not sure how their linting is set up.

Reply:
You should get a warning if you try to import from waffle.